src/app/cli/interpreter.cpp

```diff
@@ -197,7 +197,9 @@ const std::map<std::string, Interpreter::Evaluator> &Interpreter::mEvaluatorMap
 };
 
 const std::map<std::string, std::string> &Interpreter::mUsageMap = *new std::map<std::string, std::string>{
-    {"config", "config get pskc\n"
+    {"config", "config get accesscode\n"
+               "config set accesscode <9-digits>\n"
```

Comment:
not sure if I have some misunderstanding here, but get/set accesscode seems somewhat confusing to me, for which people might think it to get/set accesscode of border router. What about adding it as a parameter to get epskc? e.g., config get epskc -c <9-digits accesscode>?

Reply:
updated with more friendly hints, pls take a look at the new patch